case-lib/lib.sh

```diff
@@ -802,6 +802,10 @@ set_alsa_settings()
 {
     # ZEPHYR platform shares same tplg, remove '_ZEPHYR' from platform name
     local PNAME="${1%_ZEPHYR}"
+
+    # Initialize alsa settings first
+    alsactl init
+
```

Comment:
@fredoh9  Note, this should also get the UCM defaults, but it is worth double-checking this does happen (alsactl also has "  -D,--ucm-defaults  execute also the UCM 'defaults' section" option but AFAIK this should not be needed)

Example default mixer settings for SOF HDA cards are here in alsa-ucm:
https://github.com/alsa-project/alsa-ucm-conf/blob/master/ucm2/Intel/sof-hda-dsp/sof-hda-dsp.conf

The "True.BootSequence" sections should be executed to put the card into a sane default state. 

FYI @greg-intel @marc-hb